intake_stac/catalog.py

```diff
@@ -165,6 +167,57 @@ class StacCollection(StacCatalog):
     name = 'stac_catalog'
     _stac_cls = pystac.Collection
 
+    def get_asset(
+        self,
+        asset,
+        storage_options=None,
+        merge_asset_storage_options=True,
+        merge_asset_open_kwargs=True,
+        **kwargs,
+    ):
+        r"""
+        Get a datasource for a collection-level asset.
+
+        Parameters
+        ----------
+        asset : str, optional
```

Comment:
asset is required right? see note below. looking at this function i got confused by `asset` and `asset_`, so could consider changing on of them. `key` or `asset_name` ?